src/collision.cpp

```diff
@@ -499,7 +498,6 @@ collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
 				info.type = COLLISION_NODE;
 
 			info.node_p = nearest_info.position;
-			info.bouncy = bouncy;
```

Comment:
Why were these removed? `bouncy`, `collides_xz` and `standing_on_unloaded` might be used in the future, if they aren't right now.